package.json

```diff
@@ -11,7 +11,7 @@
         "testsOnly": "bash run-tests.sh -s=true",
         "lint": "eslint src tests-legacy && prettier --write src tests-legacy",
         "lint:fix": "eslint src tests-legacy --fix && prettier --write src tests-legacy",
-        "prestart": "npm-link-shared ./node_modules/@exabyte-io/cove.js/node_modules . react",
+        "prestart": "npm-link-shared ./node_modules/@exabyte-io/cove.js/node_modules . react; npm-link-shared ./node_modules/@exabyte-io/wave.js/node_modules . react",
```

Comment:
This was added for local development to include wave and not have issues with two reacts